test/kubernetes/e2e/features/basicrouting/edge_suite.go

```diff
@@ -117,5 +123,51 @@ func (s *edgeBasicRoutingSuite) TestBasicVirtualServiceRouting() {
 			StatusCode: http.StatusOK,
 			Body:       gomega.ContainSubstring(testdefaults.NginxResponse),
 		})
+}
+
+func (s *edgeBasicRoutingSuite) TestVirtualServiceWithRetries() {
```

Comment:
I don't totally get what we gain from this e2e test. Seems like it's asserting that the unit tested values are passed through all the way, but they're translated with other config that's checked in plenty of other tests, right? What does this test add? I assume it's unreasonably involved to check that the backoff is _actually_ the right interval in a case with a response with the given headers, so maybe an e2e test isn't worthwhile in this case?